builder/bundles.go

```diff
@@ -178,6 +180,7 @@ func resolveFilesForBundle(bundle *bundle, repoPkgs repoPkgMap, packagerCmd []st
 		if err != nil {
 			return err
 		}
+		log.Debug(log.Dnf, outBuf.String())
```

Comment:
Lets have this debug statement inside the if loop so that we don't clutter the logs with dnf output unless there is an error.
E.g.
```
		if err != nil {
			log.Debug(log.Dnf, outBuf.String())
			return err
		}
```

Reply:
should we not print the dnf output in debug, as debug mode gives more information of the output

Reply:
I think it will be good idea to print the debug output of commands even if there is no error.

Reply:
I wouldn't mind that.. but if you check the log, dnf is writing a lot to the output and debug is out default log level. That is why I feel we could just provide more debug info only when there is an error.
But I could go both ways. 
Lets ask @phmccarty

Reply:
I think that logging output of every DNF command, whether it succeeded or not, is going to be far too verbose to be useful. If mixer had a log level "MAXDEBUG" or something like that, it might be okay at that level.

Reply:
we already have log levels corresponding to error, warning , info, debug and verbose. 
so will it be a good idea to have dnf output in debug mode?
If user doesn't want that detailed output, they can set it to error, warning or info mode.

Reply:
Lets define the verbose function like others:

```
// Verbose prints a verbose log entry with VRB tag
func Verbose(cmdTag, format string, a ...interface{}) {
	if level < LogLevelVerbose || !logging {
		return
	}
	if _, ok := cmdMap[cmdTag]; !ok {
		cmdTag = Mixer
	}
	logTag("VRB", cmdTag, format, a...)
}
```

and wherever there are dnf calls, lets use this verbose function. E.g.
```
			rpm, e := repoPkgFromNoopInstall(outBuf.String())
			log.Verbose(log.Dnf, errStr.Error())
			log.Verbose(log.Dnf, outBuf.String())
			if len(bundle.AllPackages) != 0 && e != nil {
				e = errors.Wrapf(e, bundle.Name)
				log.Error(log.Dnf, e.Error())
				if log.GetLogLevel() == log.LogLevelDebug {
					log.Debug(log.Dnf, errStr.Error())
					log.Debug(log.Dnf, outBuf.String())
				}
				errorCh <- e
				return
			}
```
Define log.GetLogLevel() as:
```
// GetsLogLevel returns the log level
func GetLogLevel() (l int) {
	return level
}
```